ittapi-rs/build.rs

```diff
@@ -0,0 +1,59 @@
+#![allow(unused_imports)]
+use cmake;
+use cmake::Config;
+use std::env;
+use std::path::PathBuf;
+extern crate bindgen;
+
+#[cfg(target_os = "windows")]
+fn main() {}
```

Comment:
Do you plan to add support for Windows?

Reply:
I am not setup to test that so I was going to leave that support out of this initial push. Definitely would be ideal though.

Reply:
Will work on for future edition and not lack of current support in the README on next push.